pkg/cmd/docgen/diagrams.go

```diff
@@ -717,14 +726,17 @@ var specs = []stmtSpec{
 		name:   "explain_stmt",
 		inline: []string{"explain_option_list"},
 		replace: map[string]string{
-			"explain_option_name": "( 'VERBOSE' | 'TYPES' | 'OPT' | 'DISTSQL' )",
+			"explain_option_name": "( 'VERBOSE' | 'TYPES' | 'OPT' | 'DISTSQL' | 'VEC' )",
 		},
 		exclude: []*regexp.Regexp{regexp.MustCompile("'ANALYZE'")},
 	},
 	{
-		name:  "explain_analyze_stmt",
-		stmt:  "explain_stmt",
-		match: []*regexp.Regexp{regexp.MustCompile("ANALYZE")},
+		name: "explain_analyze_stmt",
+		stmt: "explain_stmt",
+		match: []*regexp.Regexp{
+			regexp.MustCompile("ANALYZE"),
+			regexp.MustCompile("ANALYSE"),
```

Comment:
Only `ANALYZE` shows up above in `explain_stmt`. It seems like these should be the same?